velox/substrait/SubstraitToVeloxPlanValidator.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/substrait/SubstraitToVeloxPlan.h"
+
+namespace facebook::velox::substrait {
+
+/// This class is used to validate whether the computing of
+/// a Substrait plan is supported in Velox.
+class SubstraitToVeloxPlanValidator {
+ public:
+  /// Used to validate whether the computing of this type is supported.
```

Comment:
There is a lot of duplication in these comments. Perhaps, shorten to `Validate the type`

Reply:
Fixed.